.travis.yml

```diff
@@ -11,11 +11,4 @@ install:
   - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then python setup.py install; fi
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then python setup.py install; fi
 script:
-  - cd test
-  - python test_utils.py
-  - python test_method.py
-  - python test_parser.py
-  - python test_io.py
-  - python test_data.py --status --terse
-  - cd ../data && bash regression_download.sh
-  - cd ../test && python run_regressions.py --status --traceback
+  - sh run_tests.sh
```

Comment:
I should have been more clear. Since I always run the same Travis commands myself when I'm developing, I just moved them to a script. This way, anyone can do `$ sh ./run_tests.sh`. You do run the tests before you commit, don't you? 😉 I'll add a comment in the shell script and update the PR comment, because I think this is more general than Travis.